monkey/common/events/credentials_stolen_events.py

```diff
@@ -16,3 +17,6 @@ class CredentialsStolenEvent(AbstractAgentEvent):
     """
 
     stolen_credentials: Sequence[Credentials] = Field(default_factory=list)
+
+    class Config(SecretEncodingConfig):
+        pass
```

Comment:
Why is this change needed? What do we not want from `InfectionMonkeyModelConfig`?

Reply:
I guess I'm saying we should also inherit `InfectionMonkeyModelConfig`